scripts/create-package/plop-templates-storybook/demo/index.tsx.hbs

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
remove this. we added this for button for testing ie11, but it's not needed for all (at least not yet)